scripts/generate-releases-manifest.mjs

```diff
@@ -127,7 +127,7 @@ const releasesData = JSON.parse( await fs.readFile( releasesPath, 'utf8' ) );
 if ( isDevBuild ) {
 	console.log( 'Overriding latest dev release ...' );
 
-	if ( ! commitCount ) {
+	if ( commitCount !== 0 && ! commitCount ) {
```

Comment:
Hm, is this check correct?
What I see - getCommitCount returns number, new check allows only `>0`, but second check disallows any number more than `0`.

Is it typo and you wanted to use `=== 0`? If so, then it's weird that `! commitCount` foesn't include it...

Hm, could you please explaine and probably we need to add some comment to teh code.

Reply:
I think so, it follows the same logic as the one we have on the https://github.com/Automattic/studio/blob/trunk/scripts/prepare-dev-build-version.mjs#L14

I did notice however, that we no longer need the `commitCount` here, so I cleanup the file I bit.
Let me know if it's better now.